src/helper/browsers/browserManager.ts

```diff
@@ -0,0 +1,19 @@
+import {LaunchOptions, chromium, firefox, webkit} from '@playwright/test';
+
+const options: LaunchOptions = {
+  headless: false
```

Comment:
What if I want to run tests on headful mode for local run, but on headless mode on CI/CD pipeline?

Reply:
Fixed